AppConnectSwift/UIViewController+Extensions.swift

```diff
@@ -4,13 +4,13 @@ import UIKit
 extension UIViewController {
     // Extension method to easily access the shared, UI thread datastore that is stored in the AppDelegate
     func UIThreadDatastore() -> MDDatastore! {
-        return (UIApplication.sharedApplication().delegate as! AppDelegate).UIDatastore!
+        return (UIApplication.shared.delegate as! AppDelegate).UIDatastore!
```

Comment:
Just an observation: It's funny that for UIApplication, `shared` is a property but for NSApplication it's a `shared()` method.